src/data/ACTIONS/root/RDM.ts

```diff
@@ -82,6 +82,10 @@ export const RDM = ensureActions({
 		name: 'Grand Impact',
 		icon: iconUrl(3243),
 		potency: 600, // Gives 3 white, 3 black
+		onGcd: true,
+		cooldown: 2500,
+		castTime: 2000,
```

Comment:
Looking at the job guide, this is an instant.

Reply:
Whoops, you're right. Patching this now.

Reply:
Fixed & added myself as a contributor. With any luck, git will be smart enough to recognize that the png was already in the repo history, rather than creating another binary entry.